includes/discovery/sensors/fanspeeds/linux.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+$oids = snmp_walk($device, '.1.3.6.1.4.1.10876.2.1.1.1.1.3', '-OsqnU', 'SUPERMICRO-HEALTH-MIB');
```

Comment:
Supermicro is a bit of an oddball because it should be supported on windows too, but it probably shouldn't be it's on os.  Perhaps move to supermicro.inc.php and include in windows and linux?